project/Dependencies.scala

```diff
@@ -29,7 +29,6 @@ object Dependencies {
 
   lazy val caseInsensitive = "org.typelevel"           %% "case-insensitive"        % Versions.caseInsensitive
   lazy val cats = "org.typelevel"                      %% "cats-core"               % Versions.cats
-  lazy val catsEffectKernel = "org.typelevel"          %% "cats-effect-kernel"      % Versions.catsEffect
```

Comment:
PR #729 will need kernel, so maybe don't remove it here to avoid merge problems.